kivy/tests/test_mouse_hover_event.py

```diff
@@ -0,0 +1,224 @@
+from kivy.input.providers.mouse import MouseMotionEventProvider
```

Comment:
Actually, it's probably better to not import these globally but rather where you need it; that could be responsible for the failed tests.

Reply:
Failing rtd is because of `compoundselection.py` and that should be fixed by https://github.com/kivy/kivy/pull/7441 which is merged in master. 

The import of `MouseMotionEventProvider` is in global scope in `test_mouse_multitouchsim.py` too, but I'll move it in local scope for both modules in the next commit.